tests/src/gtest/ut_gtest_main.cpp

```diff
@@ -0,0 +1,111 @@
+#include <gtest/gtest.h>
```

Comment:
this  isn't `ut_test_main.cpp` this is a testing suite that tests stuff.


`ut_test_main.cpp` - is the code that accepts switches like `-p <profile.xml> -l <log_directory>`